### PR TITLE
use less time and space in MultiThreadedDBTest.MultiThreaded test

### PR DESCRIPTION
**Pull request by ajkr**
Previously it consumed 250MB space and took 5 minutes when run serially. Now it consumes ~10MB and takes 30 seconds. The number of files/compactions is more now, too. In the longer term we should get rid of this test and just rely on `db_stress`.

### CODE REVIEW

File: db/db_test.cc [hunk 1]
```diff
@@ -1927,7 +1927,7 @@ namespace {
 
 static const int kColumnFamilies = 10;
 static const int kNumThreads = 10;
-static const int kTestSeconds = 10;
+static const int kTestSeconds = 1;
```
Comment (siying) [inline db/db_test.cc]: Are you sure we want to reduce test time?

Comment (ajkr) [reply db/db_test.cc]: Yes, in fact I think we should delete this test entirely as it should be covered by `db_stress`. But that's a longer-term thing as we need to gain more confidence in `db_stress` first.

Comment (yuslepukhin) [reply db/db_test.cc]: But db_stress is not a test but utility?

Comment (ajkr) [reply db/db_test.cc]: That's true, though this test seems to be looking for problems in a brute force way, whereas our other tests cover particular features or edge cases. `db_stress` is dedicated to that purpose, and we run it continuously. The main reason, though, is travis keeps showing out-of-space errors so I want to get rid of the space-heavy unit tests :/.
